bokeh/models/glyphs.py

```diff
@@ -893,6 +926,39 @@ class Text(Glyph):
     The %s values for the text.
     """)
 
+class VBar(Glyph):
+    """ Render vertical bars, given a center coordinate, width and (top, bottom) coordinates. """
+
+    __example__ = "tests/glyphs/VBar.py"
+
+    # a canonical order for positional args that can be used for any
+    # functions derived from this class
+    _args = ('x', 'width', 'top', 'bottom')
+
+    x = NumberSpec(help="""
+    The x-coordinates of the centers of the bars.
+    """)
+
+    width = NumberSpec(help="""
+    The widths of the vertical bars.
+    """)
+
+    bottom = NumberSpec(help="""
+    The y-coordinates of the bottom edges.
+    """)
```

Comment:
Ah, I see. My apologies for the previous comment - I had a grammatical breakdown.

I was thinking about a particular case that isn't assumed with the default `bottom=0`:

    plot.vbar(x=[1, 2, 3], width=0.5, bottom=[-3, 0, 0], top=[0, 2, 3])

In this case, there is one negative bar. A user would have to explicitly specify that the bottom is negative, then.

Ok, on thinking about it, I think having a default `bottom=0` makes sense. I'll add this in, and likewise for HBar (left=0).